src/mono/mono/metadata/icall.c

```diff
@@ -2532,15 +2532,15 @@ method_is_reabstracted (MonoMethod *method)
 	/* only on interfaces */
 	/* method is marked "final abstract" */
 	/* FIXME: we need some other way to detect reabstracted methods.  "final" is an incidental detail of the spec. */
-	return method->flags & METHOD_ATTRIBUTE_FINAL && method->flags & METHOD_ATTRIBUTE_ABSTRACT;
+	return m_method_is_final (method) && m_method_is_abstract (method);
```

Comment:
Is there a scenario where `method->flags` is NULL?

Reply:
`method->flags` is an integer, so it's not null.  `method` might be null in general, but I'm not sure if `method` (which comes from an interface) is ever null, or if `foundMethod` (which comes from a class implementing that interface) is ever null.

Reply:
Right, sorry, meant method is NULL. It's early 🙂